es-models/gdc_from_graph/annotation.mapping.yaml

```diff
@@ -12,68 +12,95 @@ properties:
         analyzer: autocomplete_prefix
         search_analyzer: lowercase_keyword
         type: text
+    normalizer: clinical_normalizer
     type: keyword
   annotation_id:
-    copy_to:
-    - annotation_autocomplete
+    copy_to: annotation_autocomplete
+    normalizer: clinical_normalizer
     type: keyword
   case_id:
+    normalizer: clinical_normalizer
     type: keyword
   case_submitter_id:
+    normalizer: clinical_normalizer
```

Comment:
Should we also be skipping the normalizer on these fields like `case_submitter_id`, `entity_submitter_id`, etc.?

Reply:
good catch, we probably should.